Python/fileutils.c

```diff
@@ -77,7 +77,9 @@ _Py_device_encoding(int fd)
     if (!valid)
         Py_RETURN_NONE;
 
-#if defined(MS_WINDOWS)
+#ifdef MS_WINDOWS_NON_DESKTOP
+    Py_RETURN_NONE;
```

Comment:
```suggestion
#ifdef MS_WINDOWS_GAMES
    Py_RETURN_NONE;
```
The console API is supported by apps.